plugin/evm/validators/manager.go

```diff
@@ -145,7 +145,7 @@ func loadValidators(validatorState stateinterfaces.State, newValidators map[ids.
 			Weight:         newVdr.Weight,
 			StartTimestamp: newVdr.StartTime,
 			IsActive:       newVdr.IsActive,
-			IsSoV:          newVdr.IsSoV,
+			IsL1Validator:  newVdr.IsSoV,
```

Comment:
Can/should we go to the latest commit of AvalancheGo where this is updated to [`IsL1Validator`](https://github.com/ava-labs/avalanchego/blob/master/snow/validators/validator.go#L41) also?

Reply:
https://github.com/ava-labs/subnet-evm/pull/1389